aggie-coding-club/Rev-Registration
# Frontend/save schedules

## Description

Adds functionality to save/unsave and remove schedules.

## Rationale

A couple of potentially controversial/questionable changes I made:
- Colors for save/delete buttons are the same because I wasn't sure what color if any to change them to, if anyone has suggestions then go ahead
- A couple tests are causing a minor error I'm not sure how to fix, I'll post a comment describing this in more detail

## How to test

Just generate schedules and save them, you can see that when you click generate schedules again it keeps your saved schedules.

## Screenshots

|Before|After|
|--|--|
|![image](https://user-images.githubusercontent.com/28655462/84580219-94b9e100-ad9a-11ea-976d-646c2298d5e5.png)|![image](https://user-images.githubusercontent.com/28655462/86314811-38eeb500-bbee-11ea-8b0b-fdb32b0581a7.png)|

## Related tasks

Closes #228.
